<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <property name="state" stdset="0">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">#InIPAddress[State = &quot;0&quot;]{
background-color: rgb(234,0,0);
}

#InIPAddress[State = &quot;1&quot;]{
background-color: rgb(67,135,100);
}</string>
      </property>
      <property name="title">
       <string>Connection information</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>IP Address</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lnIPAddress">
         <property name="styleSheet">
          <string notr="true">#lnIPAddress[state = &quot;0&quot;]{
	background-color: rgb(255, 0, 0)
}

#lnIPAddress[state = &quot;1&quot;]{
background-color: rgb(0, 85, 0)
}</string>
         </property>
         <property name="inputMask">
          <string>000.000.000.000;_</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
         <property name="state" stdset="0">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Port:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spnPort">
         <property name="minimum">
          <number>1024</number>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>12345</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="btnConnect">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="lstConsole"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnClear">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="grpSendData">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Send Data</string>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Message</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lnMsg">
         <property name="styleSheet">
          <string notr="true">#lnIPAddress[state = &quot;0&quot;]{
	background-color: rgb(255, 0, 0)
}

#lnIPAddress[state = &quot;1&quot;]{
background-color: rgb(0, 85, 0)
}</string>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
         <property name="state" stdset="0">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btnSend">
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>565</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClear</sender>
   <signal>clicked()</signal>
   <receiver>lstConsole</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
